apps/nauticalflags/nauticalflags.star

```diff
@@ -17,11 +17,11 @@ flags = {
AgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAKPD94cGFja2V0IGVuZD0idyI/PqNMOSkAAAAJcEhZcwAALiMAAC4jAXilP3YAAAAHdElNRQfoBxUWKjMrNDoOAAAAGXRFWHRDb21tZW50AENyZWF0ZWQgd2l0aCBHSU1QV4EOFwAAAcRJREFUWMPF18tLW0EUwOHf3CTG3Ehb0JqIIgpdhEI23bjzvVGhoLRiqULx7aboptB/QVyIgqIUfARsBVEKoq7cFBoJCqW0IGhAoivFICSoN7F33AlFbUG4M7M8c4b5hjkwcwQg0TjcAFI+3OAt63nQuhxxRfpgFkPXyV+UXwLoARTmXvDxwzs9AI8hedVQSlVVpXqABIrzUnR3v8Xn86kHPHL/YXCgjlAodBNTBhBAxXNBc/NLXC6XesATzwUDvS0Eg8G/4koALiFpbSyhtrbm1pwSwLP8S7o632CapnpAnsems72CcDh857zjgHCpRVvbawzDUA94mmsx9L6VQCBwb45jAI8BTdUF1NfXIYRQDyjyp+nv68Dv9/8zzzFA9ir7380dBZxaj5mZWcCyLD2AjG0QWd4jGo3qAQCcWCajY/Mkk0k9AICt35KVla/3fvscB5xZbj7NbhKPx/UAJPDjMIf5yOKdBankMbIxWFjeJRaL6QEAHKW8TE59uVWQygASwdr3FGvrG9i2rR4AcG57GZ9YJZFI6AEIYP/YZG7uM5lMRk9fkM4KIku/2N7e0dcZHaZ8DI9M6wOA4NtPeXMtWtvzaxpshEd4/bF8AAAAAElFTkSuQmCC",
```

Comment:
```suggestion
        "flag": "iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAABxElEQVRYw8XXy0tbQRTA4d/cJMbcSFvQmogiCl2EQjbduPO9UaGgtGKpQvHtpuim0H9BXIiCohR8BGwFUQqirtwUGgkKpbQgaECiK8UgJKg3sXfcCUVtQbgzszxzhvmGOTBzBCDRONwAUj7c4C3redC6HHFF+mAWQ9fJX5RfAugBFOZe8PHDOz0AjyF51VBKVVWleoAEivNSdHe/xefzqQc8cv9hcKCOUCh0E1MGEEDFc0Fz80tcLpd6wBPPBQO9LQSDwb/iSgAuIWltLKG2tubWnBLAs/xLujrfYJqmekCex6azvYJwOHznvOOAcKlFW9trDMNQD3iaazH0vpVAIHBvjmMAjwFN1QXU19chhFAPKPKn6e/rwO/3/zPPMUD2KvvfzR0FnFqPmZlZwLIsPYCMbRBZ3iMajeoBAJxYJqNj8ySTST0AgK3fkpWVr/d++xwHnFluPs1uEo/H9QAk8OMwh/nI4p0FqeQxsjFYWN4lFovpAQAcpbxMTn25VZDKABLB2vcUa+sb2LatHgBwbnsZn1glkUjoAQhg/9hkbu4zmUxGT1+QzgoiS7/Y3t7R1xkdpnwMj0zrA4Dg2095cy1a2/NrGmyER3j9sXwAAAAASUVORK5CYII=",
```

Can shrink these even more by stripping EXIF data: `$ exiftool -all= file.png`. Reduces this one from 6195 to 509 bytes.